src/components/switch.re

```diff
@@ -3,9 +3,13 @@
 let make =
     (
       ~disabled: option(bool)=?,
+      ~trackColorTrue: option(string)=?,
+      ~trackColorFalse: option(string)=?,
+      ~ios_backgroundColor: option(string)=?,
```

Comment:
I know that this is the RN name, but what about `iosBackgroundColor` ? The mix of _ & uppercase is a bit awkward to me.

Reply:
You're right, looks quite ugly. I've revised accordingly.